react/Layout/Layout.jsx

```diff
@@ -59,20 +53,16 @@ export const Layout = forwardRef(
 Layout.displayName = 'Layout'
 
 export const Main = forwardRef(({ className, children, ...props }, ref) => {
+  const { isDesktop } = useBreakpoints()
   const { monoColumn, withTopBar } = useLayout()
-  const variant = monoColumn ? 'monocolumn' : '2panes'
 
   return (
     <main
       ref={ref}
-      className={cx(
-        className,
-        styles['o-layout-main'],
-        styles[`o-layout-main-${variant}`],
-        {
-          [styles[`o-layout-main-${variant}--topbar`]]: withTopBar
-        }
-      )}
+      className={cx(className, styles['o-layout-main'], {
+        [styles['o-layout-main-2panes']]: !monoColumn,
+        [styles[`o-layout-main-topbar`]]: withTopBar && !isDesktop
```

Comment:
Sure about the `!isDesktop` ? If this class is responsible of moving the content down of the size of the topbar we need it in desktop too?